qiskit/providers/models/backendconfiguration.py

```diff
@@ -486,9 +486,15 @@ def from_dict(cls, data):
         Returns:
             GateConfig: The GateConfig from the input dictionary.
         """
-        gates = [GateConfig.from_dict(x) for x in data.pop('gates')]
-        data['gates'] = gates
-        return cls(**data)
+        in_data = copy.copy(data)
+        gates = [GateConfig.from_dict(x) for x in in_data.pop('gates')]
+        in_data['gates'] = gates
+        input_uchannels = in_data.pop('u_channel_lo')
+        u_channels = []
+        for channel in input_uchannels:
+            u_channels.append([UchannelLO.from_dict(x) for x in channel])
```

Comment:
So the type hint at line 363 is wrong. `u_channel_lo` should be `List[List[UchannelLO]]`.

Reply:
Fixed in: https://github.com/Qiskit/qiskit-terra/pull/4124/commits/14ca7e107e8fa0da9a72aa03b30353c41322b2de